MovieApp/MovieApp/FeedTableViewCell.h

```diff
@@ -0,0 +1,19 @@
+//
+//  FeedTableViewCell.h
+//  MovieApp
+//
+//  Created by user on 23/09/16.
+//  Copyright © 2016 internshipABH. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "CustomTableViewCell.h"
+
+@interface FeedTableViewCell : CustomTableViewCell
+
+@property (weak, nonatomic) IBOutlet UILabel *labelHeadline;
+@property (weak, nonatomic) IBOutlet UITextView *textViewText;
```

Comment:
This can be replaced with `UILabel` named `textLabel`. We will display first 5 lines and by finishing paragraph with ellipsis we will indicate that there is more to read (on the web by clicking source link).